packages/mobile/src/exchange/WithdrawCeloSummary.tsx

```diff
@@ -29,7 +24,8 @@ export default function WithdrawCeloSummary({
   recipientAddress,
 }: WithdrawCeloProps) {
   const { t } = useTranslation(Namespaces.exchangeFlow9)
-  const currentAccount = useSelector(currentAccountSelector)
+  // TODO: Estimate real fee.
+  const fee = new BigNumber(0)
```

Comment:
Could we instead have this be a small non-zero number? My thinking is it's better to overestimate rather than underestimate when showing the user costs. I think this would also be good because it will ensure the user has enough CELO to pay for gas.

Reply:
I'll preface this by saying that I don't fully understand how the fees are calculated, but:

- This number is for showing on the UI only, not the real one. I tried using the `max` number and there are no fee issues. Not sure why. Withdrawing without cUSD balance also works.
- If we put something non-zero here, it would still be rounded to 0 since we only show three decimals, and whatever fee is charged is lower than that. So leaving it as 0 seems fine to me. 
- Also, the buy/sell CELO flow has a 0 too and I don't think we've received any complaints or issues from that.

Reply:
I get what you mean - the number is only an estimate for display and doesn't impact the actual transaction at all. If the buy/sell CELO flow is 0 as well, I think leaving it at 0 here makes sense. We can keep it consistent for now and change all of them if/when we decide to.